src/frontend/src/lib/components/tokens/TokenExchangeBalance.svelte

```diff
@@ -5,17 +5,15 @@
 
 	export let balance: TokenFinancialData['balance'];
 	export let usdBalance: TokenFinancialData['usdBalance'];
+	export let nullishBalanceMessage = '-';
```

Comment:
why a prop? I think you can use directly inside the component since that is its usage

Reply:
This needs to be a prop because by default, this property will display `-` when the value is null or undefined. However, for the `Hero` element, it should display `'$ value is not available'` instead.

Reply:
I am still not convinced about this, i would prefer to default is as `undefined` and the use coalescing to overwrite it with `'-'`